pmoves/services/hi-rag-gateway-v2/routes/geometry.py

```
@@ -0,0 +1,574 @@
+            nc = await _nats.connect(servers=[NATS_URL])
+            await nc.publish("mesh.shape.handshake.v1", json.dumps(payload).encode())
+            await nc.flush(); await nc.drain()
+        asyncio.run(_pub())
+        return {"ok": True}
+    except Exception as e:
+        logger.exception("mesh publish failed")
+        raise HTTPException(500, f"mesh publish failed: {e}")
+
+
+@router.post("/geometry/import_db")
+def import_db(body: Dict[str, Any], _=Depends(require_admin_tailscale)):
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)